packages/opentelemetry-core/package.json

```diff
@@ -46,6 +49,29 @@
   "engines": {
     "node": ">=14"
   },
+  "type": "module",
+  "exports": {
+    ".": {
+      "require": {
+        "types": "./build/src/index.d.ts",
+        "default": "./build/src/index.js"
+      },
+      "import": {
+        "types": "./build/esm/index.d.ts",
+        "default": "./build/esm/index.js"
+      }
+    },
+    "./platform": {
+      "require": {
+        "types": "./build/src/platform/index.d.ts",
+        "default": "./build/src/platform/index.js"
+      },
+      "import": {
+        "types": "./build/esm/platform/index.d.ts",
+        "default": "./build/esm/platform/index.js"
+      }
+    }
+  },
   "files": [
```

Comment:
This "files" should actually contain the entire build directory, or somehow ensure the `package.json` is included (i.e. also include `*.json`)